apps/desktop/desktop_native/core/src/ipc/mod.rs

```diff
@@ -82,3 +91,25 @@ pub fn path(name: &str) -> std::path::PathBuf {
         path_dir.join(format!("app.{name}"))
     }
 }
+
+/// Paths to the ipc sockets including alternative paths.
+/// For flatpak, a path per sandbox is created.
+pub fn all_paths(name: &str) -> Vec<std::path::PathBuf> {
+    #[cfg(target_os = "linux")]
+    {
+        // On Linux, in flatpak, we mount sockets in each app's sandboxed directory.
+        let user_home = dirs::home_dir().unwrap();
+        let flatpak_path = user_home.join(".var/app/");
+        let flatpak_paths = FLATPAK_PATHS
+            .iter()
+            .map(|path| flatpak_path.join(path).join(format!(".app.{name}.socket")))
+            .collect::<Vec<_>>();
```

Comment:
You don't need to collect here, extend takes an iterator already
```suggestion
            .map(|path| flatpak_path.join(path).join(format!(".app.{name}.socket")));
```